### DISCUSSION TITLE
tvanish or check for permission of vanish

### DISCUSSION THREAD

**Discussion by jeremystve** [Zrips/CMI#2056] [Feature Requests]
Hey,

I wanted to ask if you could add a tvanish command like tfly or a check which checks if the player still has the vanish permission or not :)

Thank you!

**Comment by HomeLandingsDE** [comment]
just use /cmi haspermission Playername cmi.command.vanish 

or check Info: /cmi info Playername

**Comment by jeremystve** [comment]
This actually just tells  me if he has the permission or not. The player is still vanished after he lost the permission and can not unvanish by himself (ofc :D).

My fault, I did not point out that I want him to get unvanished after he lost the permission... Thats why I need a /tvanish.
